sycl/include/sycl/info/info_desc.hpp

```diff
@@ -191,14 +191,14 @@ template <typename T, T param> struct compatibility_param_traits {};
   } /*namespace info */                                                        \
   } /*namespace Namespace */
 
-namespace ext::oneapi::experimental::info {
+namespace ext::oneapi::experimental {
 
 enum class graph_support_level { unsupported = 0, native, emulated };
```

Comment:
I think that @steffenlarsen mentioned in one of his review that it is preferred to have explicit enum. 
i.e. enum class graph_support_level { unsupported = 0, native = 1, emulated = 2 };